src/main/java/com/example/agencydataapi/service/AgencyDataService.java

```diff
@@ -44,6 +68,5 @@ private AgencyRefData mapToAgencyRefData(Map<String, Object>
     }
 
     public void saveAgencyRefData(List<AgencyRefData> agencyRefDataList) {
-        writeAgencyRefData(agencyRefDataList);
```

Comment:
this line should be recovered and do import static of writeAgencyRefData from AgencyUtils